pkg/chartutil/create_test.go

```diff
@@ -26,6 +26,14 @@ import (
 	"k8s.io/helm/pkg/proto/hapi/chart"
 )
 
+func checkFile(t *testing.T, dir string, f string) {
```

Comment:
can we refactor this into

```
func checkFile(t *testing.T, f string) {
```

Then change every function call to the full filepath? e.g.

```
checkFile(t, filepath.Join(dir, f))
```

There's no need to separate `dir` and `f` if we're just going to call `filepath.Join` a second time.

Reply:
updated